roundtripper/roundtripper.md

```diff
@@ -0,0 +1,31 @@
+## Caching
```

Comment:
usually these files can be named README.md (readme for a directory).

Reply:
Done.